org.kde.kfloppy.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "org.kde.kfloppy",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kfloppy",
+    "rename-icon": "kfloppy",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
+    ],
+
+    "modules": [
+        {
+            "name": "kfloppy",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                { "type": "git",
+                  "commit": "c98dbb8c326b380bac0b23233c9015716815923f",
```

Comment:
you need to use a tarball not a git commit

Reply:
https://github.com/flathub/flathub/pull/2749/commits/fcee4e64bf3fe206590b00ce89fc014ca88c91a4